### ISSUE TITLE
[Build] Mobile Table FE template

### ISSUE THREAD

**Issue by brianalloyd** [department-of-veterans-affairs/va.gov-team#14618]
## Issue Description
_Give Build FE Template for Content Type: About [#13963] is closed, I want to begin building the FE mobile table template so that this component is successfully used on Learning Center_

## Front end tasks
- [x] - Build Mobile Table FE Template
- [x] - Ensure no CMS dependencies exist

## Acceptance Criteria
- [ ] Ensure layer of the FE mobile table is complete for the content type
- [ ] Make sure [any guidance from the original issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/13963#issuecomment-698416821) for this work is taken into consideration
- [ ] Need to groom, review, estimate

## Planning Considerations 
Will apply Sitewide (and to the About template)

**Comment by ncksllvn** [comment]
Crosslinking to a thread about another table, https://github.com/department-of-veterans-affairs/va.gov-team/issues/6009#issuecomment-706134199

**Comment by kelsonic** [comment]
Hey @andaleliz, would you mind linking the mobile table design here for the about template if you have it handy?

**Comment by kelsonic** [comment]
CC @youngfreezyVA

**Comment by andaleliz** [comment]
[Here's the prototype](https://preview.uxpin.com/e7717d3fa019146277f28d799d247b820140d1b5#/pages/132642759/documentation/no-panels?mode=i). I added the link to the original comment where I shared these, since there are some more details about this work.  Also, see the comments after the one I link to - I think Jen Strickland shared some relevant code or further guidance.

**Comment by youngfreezyVA** [comment]
to summarize... the reason the design was not achievable in 1.0 is because the columns for the mobile layout have essentially changed, and since we could not change the entity to return a new set of mobile columns, we had to make the decision to push this new logic to the front end. in order to accommodate the new columns, we created a new `mobileColumnsData` variable in the liquid js filters.

then we ran into an issue with the additional info component - it does not handle dynamically/conditionally rendered multiple divs of content (for example the equivalent of a `.map` statement inside jsx) and expects hidden content to be a child of the additional info content container. remember, we are rendering additional info content _while we are looping through the columns_ there is no other use case of the additional info component being used like this. every other usage of additional info content renders pre defined content from the backend, using the `   entity.fieldWysiwyg.processed ` to render an html node.

here is an example:
```{% comment %}
  Example data:
  "entity": {
    "entityType": "paragraph",
    "entityBundle": "spanish_translation_summary",
    "entityId": "937",
    "fieldWysiwyg": {
      "processed": "..."
    },
    "fieldTextExpander": "Obtenga instrucciones para esta solicitud en Español. "
  }
{% endcomment %}

<div data-template="paragraphs/spanish_translation" data-entity-id="{{ entity.entityId }}" class="form-expanding-group additional-info-container">
  <span class="additional-info-title">{{ entity.fieldTextExpander }}</span>
    <span id="spanishhelptext-{{ entity.entityId }}">
      <div class="additional-info-content">
        {{ entity.fieldWysiwyg.processed }}
      </div>
    </span>
</div>
```
so in short we can go one of two routes:
- change columns returned from backend to return mobile columns as well
- make a PR to the additional info component to handle multiple content nodes that are dynamically rendered.

**Comment by andaleliz** [comment]
There is a lot here I don't understand (e.g. what is a `.map` statement? what is jsx), but I appreciate the details! I trust the team will do whatever it takes to complete this in a future version.  We'll need to make sure to relay this to QA so they're not looking for this functionality in QA.

>change columns returned from backend to return mobile columns as well

Not sure what this route means but I think the DOM should be the same between mobile and desktop, just the CSS is what is changing.

**Comment by brianalloyd** [comment]
Thanks for the comments @andaleliz and @youngfreezyVA, these are important findings that will help drive MVP 1.1 functionality so thanks for including the details.
